Cogs.Publishers/JsonSchema.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cogs.Publishers
+{
+    class JsonSchema
+    {
+        public string title;
```

Comment:
Lets convert these to get and set properties, and use camel casing. Such as

public string Title { get; set; }

We can do this in both of these classes